discovery-infra/start_discovery.py

```diff
@@ -88,12 +89,20 @@ def set_hosts_roles(client, cluster_id):
         client.set_hosts_roles(cluster_id=cluster_id, hosts_with_roles=hosts)
 
 
+def set_cluster_vips(client,cluster_id):
```

Comment:
why not to add it on create? why need to update cluster?

Reply:
because Ori is checking machine-cidr on updates, that's the original bug

Reply:
can u explain a little more?
this can be pushed on cluster create or not?
If we support passing those params on create we need to support it :)